pom.xml

```diff
@@ -68,16 +68,15 @@
     </scm>
 
     <repositories>
+        <!-- required for javax.script:groovy-engine:1.1(1.1-jdk14) -->
         <repository>
-            <id>java.net repository</id>
-            <url>http://download.java.net/maven/2</url>
+            <id>Mulesoft Repository</id>
```

Comment:
is this expected?

Reply:
Yep! Had to replace the java.net repository with Mulesoft